Workbooks/Azure Active Directory TroubleShooting/SignInFailureAnalysis/SignInFailureAnalysis.workbook

```diff
@@ -1,14 +1,13 @@
 {
-  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
   "version": "Notebook/1.0",
-  "isLocked": true,
```

Comment:
isLocked is removed. Is this intentional?

Reply:
locked:true is the default, so the workbook UI no longer exports this value at all, to avoid the cases where people were accidentally pushing templates with it set to true.

Reply:
(if someone WANTS a template to open in edit mode (rare, there's only one?), the author has to jump through the hoop of explicitly adding `"isLocked": false,` to the template)

Reply:
Got it